articles/140_topic_and_service_name_mapping.md

```diff
@@ -131,10 +131,10 @@ Topic and service names:
 
 - must not end with a forward slash (`/`)
 
-The last token is the topic or service basename, and any preceding tokens make up the namespace of the topic or service.
+The last token is the topic or service base name, and any preceding tokens make up the namespace of the topic or service.
```

Comment:
We were using `basename` and `base name` throughout, and I decided to use `base name` because the spell checker was happier with it and the snippet from ROS 1 uses that form.